metabase/metabase
# Patch zrender until a new version is released

### Description

`zrender` library used by ECharts could not apply text styles when CSP is enforced. The has been merged already https://github.com/ecomfe/zrender/pull/1030 but a new version has not been released. This is a temporary patch until it is released. 😬 

### How to verify

Try line/area/bar/combo charts and ensure they have correct font styles on axis ticks and data labels.

### Checklist

- [n/a] Tests have been added/updated to cover changes in this PR
